v2/internal/pkg/helm/local_stored_collector_test.go

```diff
@@ -539,6 +540,32 @@ func TestHelmImageCollector(t *testing.T) {
 			},
 			expectedError: nil,
 		},
+		{
+			caseName:     "local helm chart with images on deployment env vars- MirrorToDisk: should pass",
+			mirrorMode:   mirror.MirrorToDisk,
+			localStorage: testLocalStorageFQDN,
+			helmConfig: v2alpha1.Helm{
+				Local: []v2alpha1.Chart{
+					{Name: "test-mirror-helm", Path: filepath.Join(testChartsDataPath, "test-mirror-helm-0.3.0.tgz"), ImagePaths: []string{"{.spec.template.spec.containers[*].env[*].value}"}},
```

Comment:
I am OK to include this helm chart provided earlier, as part of the test articacts